resources/views/pages/checkout.blade.php

```diff
@@ -59,14 +59,7 @@
                                     <td class="product-quantity">{{ $row->options->size }}</td>
                                     @endif
                                     <td>
-                                        <form method="post" action="/update/cart/item">
-                                            @csrf
-                                            <input type="hidden" name="productid" value="{{ $row->rowId }}">
-                                            <input type="number" name="qty" value="{{ $row->qty }}"
-                                                style="width: 50px;" min="1">
-                                            <button type="submit" class="btn btn-success btn-sm"><i
-                                                    class="fas fa-check-square"></i>✔</button>
-                                        </form>
+                                          @include('forms.quantity_update')
```

Comment:
Move this to the the `partials`  and adjust the @include

Reply:
Corrections made.